docs/source/conf.py

```diff
@@ -21,6 +21,7 @@
     "numpydoc",
     "nbsphinx",
     "sphinx_gallery.gen_gallery",
+    "sphinxemoji.sphinxemoji",
```

Comment:
What is this for?

Reply:
This is for emojiis like `:white_check_mark:` as used in the `gpu.rst` file